src/appshell/ieditmenucontroller.h

```diff
@@ -0,0 +1,32 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2021 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_APPSHELL_IEDITMENUCONTROLLER_H
+#define MU_APPSHELL_IEDITMENUCONTROLLER_H
+
+#include "modularity/imoduleexport.h"
+#include "uicomponents/imenucontroller.h"
+
+namespace mu::appshell {
+class IEditMenuController : public uicomponents::IMenuController, MODULE_EXPORT_INTERFACE
```

Comment:
You can create a register for IMenuController instead of creating many empty interfaces and use it:
![изображение](https://user-images.githubusercontent.com/67867444/110692716-52211a00-81ef-11eb-9815-e5df39b5ddfe.png)